recipes/smif/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "smif" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "a806ef66b785e1620300bc9ce55a5488fc77d393ea3175db661cc741e234acf0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - smif = smif.cli:main
+
+requirements:
+  build:
```

Comment:
Does the `build` need `six`?:

https://github.com/nismod/smif/blob/master/setup.py#L18

Reply:
Yes, I think that should be in there.